src/anndata/tests/helpers.py

```diff
@@ -268,15 +280,60 @@ def maybe_add_sparse_array(
     return mapping
 
 
+# fixing the ndarray issue
+def _is_sparse_like(x):
+    try:
+        import scipy.sparse as sp
+
+        return sp.issparse(x)
+    except ImportError:
+        return False
```

Comment:
you should use `isinstance` with the `CS*` types in `anndata.compat` instead of this function

```suggestion
```